### ISSUE TITLE
Add FLST support in Collections

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#78]
Killer app for LoTD users who don't use Relic Notifications. FLST.EDID is not loaded - will need to specify plugin and a disambiguating list of Item Names - then scan the plugins' FLSTs for a unique match on all Named Items.

**Comment by SteveTownsend** [comment]
This item suffers from the same old 'idk what this is so I shall call it clutter' issue. In the 1.5 revision there will be a couple ways to prevent this. 
1. make it part of a Collection and don't auto-loot the members, or do loot them - at least you are getting what you affirmatively want then. Collections can be glowed as well as auto-looted, or set to 'do not pick up' if total immersion is your thing but you want to track the Collection. 
2. configure Valuable Item Threshold to some sensible number (the stone is worth several thousand septims) and do not auto-loot valuables: glow them, or prevent pickup - the same options as with collections. 
In short, I think v1.5 helps a lot with this type of thing, depending on player preference.

**Comment by SteveTownsend** [comment]
this is done - script support for more out of the box collections required

**Comment by SteveTownsend** [comment]
Fixed in 1.5.0.0
